client/src/views/NewProject.vue

```diff
@@ -1,40 +1,43 @@
 <template>
   <html>
-    <section id="new" class="main">
+    <head>
+      <link
+        href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&family=Comfortaa:wght@700&family=Lato&family=Patrick+Hand&display=swap"
+        rel="stylesheet"
+      />
+    </head>
+    <section class="main">
       <form>
         <h1 class="title is-1">
           New Project
         </h1>
         <div class="field">
           <label class="label">Project Name:</label>
           <div class="control">
-            <input class="input" type="text" v-model="form.projectName" />
+            <input class="input" type="text" size = "40" v-model="form.projectName" />
           </div>
         </div>
         <div class="field">
           <label class="label">Description:</label>
           <div class="control">
-            <input class="input" type="text" v-model="form.description" />
+            <textarea rows="4" cols="50" v-model="form.description"></textarea>
           </div>
         </div>
         <div class="field">
           <label class="label">Links:</label>
           <div class="control">
-            <input class="input" type="text" v-model="form.links" />
+            <input class="input" type="text" size = "75" v-model="form.links" />
           </div>
         </div>
         <div class="field">
-          <div class="control">
-            <label>
-              <input type="checkbox" value="yes" v-model="form.look" />
-              Looking for New Members?
-            </label>
-          </div>
+          <label>
+            <input type="checkbox" value="yes" v-model="form.lookingForNewMembers" />
+            Looking for New Members?
+          </label>
         </div>
         <input
-          class="button is-primary margin-bottom"
+          class="btn btn-primary margin-bottom"
           type="submit"
-          @click.prevent="toSubmit"
```

Comment:
This disables the ability to submit the form by pressing enter (we want this). Implement the necessary code to prevent users from submitting the form by pressing enter in any field.